src/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,234 @@
+import React, { CSSProperties, forwardRef, RefObject } from 'react';
+import { DialogOverlay, DialogContent } from '@palmetto/dialog';
+import classNames from 'classnames';
+import { DimensionSize, CssDimensionValue } from '../../types';
+import { WIDTH_OPTIONS } from '../../lib/tokens';
+import { Box } from '../Box/Box';
+import { Icon } from '../Icon/Icon';
+import styles from './Drawer.module.scss';
+
+export type DrawerPlacementType = 'left' | 'right' | 'top' | 'bottom';
+
+export const DRAWER_PLACEMENT: DrawerPlacementType[] = ['right', 'left', 'top', 'bottom'];
```

Comment:
This export is only used internally for our testing/storybook it seems. Might be best to just move it out of the component altogether unless we want to export it to consumers.